google-beta/resource_container_cluster.go

```diff
@@ -738,6 +744,10 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		Autoscaling:    expandClusterAutoscaling(d.Get("cluster_autoscaling"), d),
 		MasterAuth:     expandMasterAuth(d.Get("master_auth")),
 		ResourceLabels: expandStringMap(d, "resource_labels"),
+		NetworkConfig: &containerBeta.NetworkConfig{
+			EnableIntraNodeVisibility: d.Get("enable_intranode_visibility").(bool),
+			ForceSendFields:           []string{"Enabled"},
```

Comment:
I don't think this ForceSendFields is working as intended. There is no `Enabled` in the `EnableIntraNodeVisibility` object so it is ignored. However since this is in Create and `false` is the default there is no need to force send a falsey value.